website/source/docs/providers/vsphere/r/virtual_machine.html.markdown

```diff
@@ -71,6 +72,9 @@ The `windows_opt_config` block supports:
 * `domain_user` - (Optional) User that is a member of the specified domain.
 * `domain_user_password` - (Optional) Password for domain user, in plain text.
 
+<a id="disks"></a>
+## Disks
+
```

Comment:
I decided to add this since it was missing (it's being linked to in line 48, but clicking that link wouldn't do much).